lora/lorawan/adaptor.go

```diff
@@ -52,13 +52,13 @@ func SetPublicNetwork(enabled bool) {
 }
 
 // ApplyChannelConfig sets current Lora modulation according to current regional settings
-func applyChannelConfig(ch *region.Channel) {
-	ActiveRadio.SetFrequency(ch.Frequency)
-	ActiveRadio.SetBandwidth(ch.Bandwidth)
-	ActiveRadio.SetCodingRate(ch.CodingRate)
-	ActiveRadio.SetSpreadingFactor(ch.SpreadingFactor)
-	ActiveRadio.SetPreambleLength(ch.PreambleLength)
-	ActiveRadio.SetTxPower(ch.TxPowerDBm)
+func applyChannelConfig(ch region.Channel) {
```

Comment:
This doesn't need to be a pointer, but why copy it?

Reply:
`region.Channel` is now an `interface` hence the change.